### DISCUSSION TITLE
Hi I just installed the latest docker [May 18]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2157] [Discord Threads]
> 🙋‍♂️ **Zho'Kaan**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Zho'Kaan**  
#### This view from the website's front page with the screenshots:  
![image0](https://cdn.discordapp.com/attachments/975134149524476045/975134457478676550/unknown.png)  
#  
> 💁 **o1lab**  
#### Sorry, thats not available. We'd a POC during launch. It has taken more time to finish existing features.  
#  
> 💁 **o1lab**  
#### would love to know your requirement  
#  
> 🙋‍♂️ **Zho'Kaan**  
#### Sure. I was looking to build something like this (to set up a trip itinerary / calendar with friends): https://www.airtable.com/universe/expUWPN4dJ8NoZFGR/trip-planneritinerary  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/975134149524476045)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
